borg/_hashindex.c

```diff
@@ -77,6 +79,7 @@ static int hash_sizes[] = {
 
 #define HASH_MIN_LOAD .25
 #define HASH_MAX_LOAD .75  /* don't go higher than 0.75, otherwise performance severely suffers! */
+#define HASH_MAX_EFF_LOAD .85
```

Comment:
https://github.com/leventov/Koloboke/wiki/Tombstones-purge-from-hashtable:-theory-and-practice#theory-and-practice that was the table. but i guess i read it wrong.